components/nexus-security/src/main/java/org/apache/shiro/nexus5727/FixedDefaultWebSessionManager.java

```diff
@@ -36,15 +36,20 @@
 public class FixedDefaultWebSessionManager
     extends DefaultWebSessionManager
 {
+  /**
+   * Default HTTP session cookie name that Nexus uses - NXSESSIONID
+   */
+  public static final String DEFAULT_NEXUS_SESSION_COOKIE_NAME = "NXSESSIONID";
```

Comment:
Was thinking more about load-balancer/sticky-session setups where you need to give it a list of session cookie names. Might not affect most people, but is a reason to document the name change in case they need to change their settings.